Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1

```diff
@@ -0,0 +1,271 @@
+﻿function Publish-NewServiceFabricApplication
```

Comment:
Yes, they're copy-pasted.  Thanks for the info on shared task code.  At this point, there should be no need to have this code shared with any other task.